### ISSUE TITLE
Failing Travis builds

### ISSUE THREAD

**Issue by c-schuhmann** [ahezard/nds-bootstrap#156]
This is more like a question: Is there any reason why the .travis.yml has been removed? I just wondered, because according to the README, you should look after travis first, if you want to compile the project yourself.

**Comment by Robz8** [comment]
It wouldn't compile with the latest devkitArm.

**Comment by piratesephiroth** [comment]
Moving get_io_dsisd to libnds made nds-bootstrap hate it.

We can build if we use commit ddd0ec6 of libnds and commit e254127 of libfat.

Also I had to add a definition of DEVICE_TYPE_DSI_SD to \cardengine_arm7\source\sdmmc.h.
(obviously this is a crappy fix and it most likely requires changing the order of stuff in some makefile)

**Comment by ahezard** [comment]
Should be fixed on master now

**Comment by ahezard** [comment]
Hum no it is still failing, but Master branch is now compiling ok with latest devkitArm and libnds

**Comment by ahezard** [comment]
rake aborted!
No Rakefile found (looking for: rakefile, Rakefile, rakefile.rb, Rakefile.rb)

=>Something changed about travis, this error does not seems to be related to bootstrap

**Comment by ahezard** [comment]
The travis file was removed somehow

**Comment by ahezard** [comment]
Still some issue, it is almost working
The strange things is that it is building Ok with latest devKitPro
Makefile:10: /home/travis/devkitPro/devkitARM/ds_rules: No such file or directory
make: *** No rule to make target `/home/travis/devkitPro/devkitARM/ds_rules'.  Stop.

**Comment by ahezard** [comment]
The problem should be here, for some reason the home directoty have changed
  - export DEVKITPRO=/home/travis/devkitPro

**Comment by ahezard** [comment]
The issue is reproductible, seems that devKitPro installation script is broken on linux

**Comment by c-schuhmann** [comment]
And there seems to be a problem with the ini file for the updater: [devkitpro/installer/#3](https://github.com/devkitPro/installer/issues/3)

**Comment by c-schuhmann** [comment]
Okay, Travis builds now pass on "master" and "card", since the updater.ini URL in the devkitARM updater has been updated. 

But I noticed something in the build log: At the end of the build script when the new nds-bootstrap.nds should be uploaded to the two servers, there is an [error](https://travis-ci.org/ahezard/nds-bootstrap/builds/342907295#L1377) while trying to copy (rename) the nds-bootstrap.nds file:
`cp bootstrap.nds $REV_NAME` -> `cp: cannot stat ‘bootstrap.nds’: No such file or directory`
Should we change it to `cp nds-bootstrap.nds $REV_NAME`? Both FTP uploads (put) fail because the file is missing.

**Comment by ahezard** [comment]
Yes seems ok now. About the 2 servers in the after_success section, they are outdated and no more available.

**Comment by ahezard** [comment]
I updated the filename and ftp address, thanks for your input c-schuhmann
